### PR TITLE
Making options usable

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

Lots of options that were introduced recently were not accessible from the high level interface.

- [x] number of samples
- [x] container selector strategy <- currently unused #134 
- [x] traversal selector strategy 
- [x] include options in md-flex

## ~Resolved Issues~

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [x] md-flex
- [x] fixing unit tests

### CODE REVIEW

File: src/autopas/selectors/TraversalSelector.h [hunk 1]
```diff
@@ -285,7 +285,7 @@ void TraversalSelector<ParticleCell>::findFastestAbsTraversal() {
   long optimalTraversalTime = std::numeric_limits<long>::max();
   AutoPasLog(debug, "TraversalSelector: Collected traversal times:");
   for (auto &&t : _traversalTimes) {
-    AutoPasLog(debug, "Traversal {} took {} nanoseconds:", t.traversal, t.time);
+    AutoPasLog(debug, "Traversal {} took {} nanoseconds", t.traversal, t.time);
```
Comment (SteffenSeckler) [inline src/autopas/selectors/TraversalSelector.h]:
```suggestion
    AutoPasLog(debug, "Traversal {} took {} nanoseconds.", t.traversal, t.time);
```

File: src/autopas/selectors/TraversalSelector.h [hunk 2]
```diff
@@ -302,16 +302,18 @@ template <class ParticleCell>
 void TraversalSelector<ParticleCell>::findFastestMeanTraversal() {
   // choose the fastest traversal and reset timings
   // reorder measurements
-  // @todo: maybe directly save measurements in this way?
   std::unordered_map<TraversalOptions, std::vector<long>, TrivialHash> measurementsMap;
+  AutoPasLog(debug, "TraversalSelector: Collected traversal times:");
   for (auto &&t : _traversalTimes) {
+    AutoPasLog(debug, "Traversal {} took {} nanoseconds", t.traversal, t.time);
```
Comment (SteffenSeckler) [inline src/autopas/selectors/TraversalSelector.h]:
```suggestion
    AutoPasLog(debug, "Traversal {} took {} nanoseconds.", t.traversal, t.time);
```

File: src/autopas/selectors/TraversalSelector.h [hunk 3]
```diff
@@ -328,16 +330,18 @@ template <class ParticleCell>
 void TraversalSelector<ParticleCell>::findFastestMedianTraversal() {
   // choose the fastest traversal and reset timings
   // reorder measurements
-  // todo: maybe directly save measurements in this way?
   std::unordered_map<TraversalOptions, std::vector<long>, TrivialHash> measurementsMap;
+  AutoPasLog(debug, "TraversalSelector: Collected traversal times:");
   for (auto &&t : _traversalTimes) {
+    AutoPasLog(debug, "Traversal {} took {} nanoseconds", t.traversal, t.time);
```
Comment (SteffenSeckler) [inline src/autopas/selectors/TraversalSelector.h]:
```suggestion
    AutoPasLog(debug, "Traversal {} took {} nanoseconds.", t.traversal, t.time);
```

**Comment by FG-TUM** [comment]
Feature requests:
#134 
#137

**Pull request merged**